components/cam/tools/mkatmsrffile/Makefile

```diff
@@ -1,11 +1,33 @@
-include $(CASEROOT)/Macros.bluefire
+#-------------------------------------
+# Makefile for mkatmsrffile tool 
+#-------------------------------------
 
-LIBDIR=/ptmp/jedwards/SMS_D.f09_g16.B1850CN.bluefire.C.083815/lib
-LIBS = -L$(LIBDIR) -lcsm_share -lpio -lmct -lmpeu -L$(LIB_NETCDF) -lnetcdf -L$(LIB_PNETCDF) -lpnetcdf
-MODDIR=$(LIBDIR)/include
+#-------------------------------------
+#Instructions:
+# Load a compiler and the associated 
+# netcdf library. Update variables "FC"
+# and "NETCDF_PATH" to add the compiler
+# and path to netcdf home directory
+#-------------------------------------
 
+FC := ifort
+NETCDF_PATH := $(NETCDF_HOME)
 
-mkatmsrffile: mkatmsrffile.F90 
-	$(FC) $(FFLAGS) -I$(MODDIR) $< -o $@  $(LIBS) $(LDFLAGS) 
+EXE := exe_mkatmsrffile
```

Comment:
Kind of picky, but can we call this just "mkatmsrffile" or something similar? The rest of the tools seem to follow this convention (`gen_domain`, `cube_to_target`, etc).